signatures/rainbow/rainbow-compressed/param/IIIc.yaml

```diff
@@ -0,0 +1,12 @@
+name: Rainbow-IIIc
+security level:
+  classical: 217
+  quantum: 167
+  nist category: 3
+  comment: classical RBS, quantum HighRank, both recover key
+failure probability: 0
+number of operations: TODO
```

Comment:
they say they assume `2^64` signatures. I'd say this qualifies as `inf` here, because this number stems from the NIST cfp which states
> For the purpose of estimating security strengths, it may be assumed that the attacker has access to signatures for no more than 2^64 chosen messages